src/utilities.test.js

```diff
@@ -34,3 +34,66 @@ describe("getBabelLoader finds the babel loader options", () => {
     expect(getBabelLoader(configForRuleUseArray, true)).toMatchSnapshot();
   });
 });
+
+describe(`tap`, () => {
+  test(`tap with message and no options`, () => {
+    const config = { foo: 'bar' }
+
+    const mockConsole = { log: jest.fn() }
+    global.console = mockConsole
```

Comment:
This is totally fine but I think you should add a `beforeAll`/`afterAll` block inside this `describe` to save the original console and restore it, so it wouldn't affect anyone trying to use console outside of this block.

Reply:
I changed a bit the structure of the test suite for the `tap` utility and also include the before all and after all.